src/views/sidebar/components/TheSidebar.vue

```diff
@@ -166,7 +166,16 @@ export default {
       )
       return `${diff.hours()}h ${diff.minutes()}m left`
     },
-    todos () {
+    incompleteTodos () {
+      const incompleteTodos = []
+      for (let i = 0; i < this.$store.state.todos.todos.length; i++) {
+        if (!this.$store.state.todos.todos[i].completed) {
+          incompleteTodos.push(this.$store.state.todos.todos[i])
```

Comment:
This for loop can be replaced with `return this.$store.state.todos.filter(t => !t.completed);`